.github/workflows/check-pull-request.yml

```diff
@@ -42,3 +42,22 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  vulnerability-scan:
```

Comment:
A separate code checkout and `npm ci` for this adds more time to the PR build, could npm audit just be run as an additional step in the pr-validator job? (that already runs multiple steps - lint, test etc)